src/acme/ui/UserUI.java

```diff
@@ -96,7 +88,7 @@ private void buildButtonPanel() {
 
 	}
 
-	private class LoginButtonListner implements ActionListener {
+	public class LoginButtonListner implements ActionListener {
```

Comment:
Can you go ahead and stub out some more code for logging in?  I created a just with some ideas for you
https://gist.github.com/simpleman19/2253ef20be81edc13230435c3cddc1cf

Reply:
Since this is tightly coupled to the database, I added a commit that handles this functionality ([see here](https://github.com/simpleman19/acmeDSTS/commit/3d2ddc2d929baae354837281faf844e23fd56529)). This is currently on my feature branch (`feature/database-setup`) but it can be merged into dev as soon as [its pull request](https://github.com/simpleman19/acmeDSTS/pull/20) is merged.